internal/daysteps/daysteps.go

```diff
@@ -11,10 +18,60 @@ const (
 	mInKm = 1000
 )
 
+// parseTraining парсит входные данные
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	funcData := strings.Split(data, ",")
+	if len(funcData) != 2 {
+		return 0, 0, errors.New("wrong data")
+	}
+
+	steps, err := strconv.Atoi(funcData[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Ошибки лучше оборачивать
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```